src/platform/nrfconnect/CHIPDevicePlatformConfig.h

```diff
@@ -197,6 +197,8 @@
 #define CHIP_DEVICE_CONFIG_CHIP_TASK_STACK_SIZE 8192
 #endif // CHIP_DEVICE_CONFIG_CHIP_TASK_STACK_SIZE
 
+#define CHIP_DEVICE_CONFIG_MAX_EVENT_QUEUE_SIZE 64
```

Comment:
Have we determined that we need this much? I just wonder as other platforms use 25.

Reply:
nvm, I've seen the test result page.